sirepo/template/radia.py

```diff
@@ -800,21 +795,22 @@ def _parse_input_file_arg_str(s):
 
 
 def _prep_new_sim(data):
-    data.models.geometry.name = data.models.simulation.name
+    data.models.geometryReport.name = data.models.simulation.name
 
 
 def _read_h5_path(sim_id, filename, h5path, run_dir=_GEOM_DIR):
     try:
-        with h5py.File(_get_res_file(sim_id, filename, run_dir=run_dir), 'r') as hf:
-            return template_common.h5_to_dict(hf, path=h5path)
+        p = _get_sim_file(sim_id, filename, run_dir=run_dir)
+        with h5py.File(p, 'r') as f:
+            return template_common.h5_to_dict(f, path=h5path)
     except IOError as e:
         if pkio.exception_is_not_found(e):
-            pkdc(f'{filename} not found in {run_dir}')
+            pkdlog(f'file {filename} not found in {p}')
             # need to generate file
             return None
     except KeyError:
         # no such path in file
-        pkdc(f'path {h5path} not found in {run_dir}/{filename}')
+        pkdlog(f'path {h5path} not found in {p}')
```

Comment:
We've been tending to a model of messages like this now: `'h5path={} not found in file={}'. It's slightly more structured, and it uses names of variables so easier to establish context when debugging.

Reply:
7679250